tsconfig.build.json

```diff
@@ -1,9 +1,8 @@
 {
   "extends": "./tsconfig.json",
+  "include": ["./lib"],
   "compilerOptions": {
     "target": "ES2017",
-    "rootDir": "./lib",
-    "sourceMap": true,
```

Comment:
I've removed the source maps, the compiled code is already quite readable, and not minified, so I think we can omit them.